coderadar-server/coderadar-core/src/main/java/org/wickedsource/coderadar/core/configuration/CoderadarConfiguration.java

```diff
@@ -14,18 +18,23 @@
 /** Provides access to all configuration parameters of the coderadar application. */
 @Component
 @ConfigurationProperties("coderadar")
+@Data
 public class CoderadarConfiguration {
 
+  @Getter(AccessLevel.NONE)
```

Comment:
why do we need Getters and Setters on `logger` and `CONFIG_PARAM_LOG_PATTERN`?

Reply:
also, try if we can live without any setters in a `ConfigurationProperties` file. After all, the values should not be writeable at runtime.

Reply:
There are no getters or setters for the `logger` and `CONFIG_PARAM_LOG_PATTERN`. I have annotated them with `AccessLevel.NONE` explicitly so that they do not get getters and setters. I noticed `final` fields are ignored by Lombok anyway, so I've removed these lines for the `CONFIG_PARAM_LOG_PATTERN`.

As for removing the setters, that doesn't seem to work. I think that's because some kind of setter injection is performed by Spring (as I can't find any usage of these setters). Using field or constructor injection doesn't seem to work however, as no ApplicationContext is created successfuly.

Reply:
then please make the logger `final static`, too

Reply:
I've made the logger `final static` now